### DISCUSSION TITLE
Wistia Video Not Recognised

### DISCUSSION THREAD

**Discussion by FlyingQuokka** [aclap-dev/video-downloadhelper#1323] [Help and Q&A]
### Your question

There's a video here: https://porterandcompanyresearch.co/tbsew-35c that VDH doesn't recognise, that I thought you might like to add to your investigations list, so VDH can handle these types of videos in the future.

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"D:\\Programs\\Multimedia\\VideoDownloadHelper\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"74688fafc193436a"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "valid_license_message_has_been_displayed": true,
    "download_directory": "D:\\Users\\Quokka\\SpecialFolders\\Downloads",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "V:\\Memberships\\Steal Our Winners\\Editions\\2024\\2024-10"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 3,
    "successfull_dl": 96,
    "never_show_successfull_dl_message": true
}

**Comment by Mattmjs** [comment]
Looks like the video won't start until tracking protection is disabled in Firefox, then it'll play and as well as detection.

**Reply by FlyingQuokka** [reply]
The video plays fine. It's just that VDH doesn't recognise it, so I thought I'd pass it over to the developers to consider adding it to the list of videos it can deal with. That's really the only reason for my post.

**Reply by Mattmjs** [reply]
That's what I'm saying, when the video played I had detection and you should as well. But if you aren't getting detection check
anti virus & firewall are not blocking companion app.

**Reply by FlyingQuokka** [reply]
OK, I found the tracking settings and turning them worked. Thanks @Mattmjs.
